Cargo.toml

```diff
@@ -43,7 +43,8 @@ enumset = { version = "1", default-features = false }
 log = { version = "0.4", default-features = false }
 uncased = { version = "0.9.7", default-features = false }
 embedded-hal-async = { version = "1", default-features = false }
-embedded-svc = { version = "0.27", default-features = false }
+# embedded-svc = { version = "0.27", default-features = false }
+embedded-svc = { git = "https://github.com/embediver/embedded-svc.git", default-features = false }
```

Comment:
Once this PR is ready for merge _and_ the corresponding one in `embedded-svc` is merged, we should restore the original dependency, and then put a `[patch.crates-io]` entry that points to the `embedded-svc` version from GIT.